src/asm/gamepad.s

```diff
@@ -171,6 +174,48 @@ SoftReset:
 .assert * <= $cbd3
 .endif
 
+.ifdef _RANDOM_WILD_WARP
+.org $cbd3
+  jsr LoopWildWarpIndex
+  ldy $0780
```

Comment:
not sure how the rom space looks here, but you could arrange that `y` has the loop index at the end of the subroutine and avoid these 3 bytes (you've already got it in `a` at various point in the routine, though you might need two `tay`'s depending on how it's implemented)